src/inmanta/resources.py

```diff
@@ -308,8 +310,16 @@ def map_field(
                 # passing along the serialized version would break the resource apis
                 json.dumps(value, default=inmanta.util.api_boundary_json_encoder)
                 return value
+            except IgnoreResourceException:
+                raise  # will be handled in _load_resources of export.py
             except proxy.UnknownException as e:
                 return e.unknown
+            except RuntimeException as e:
+                raise WrappingRuntimeException(None, "Failed to get attribute '%s' on '%s'" % (field_name, entity_name), e)
+            except plugins.PluginException as e:
+                raise ExplicitPluginException(None, "Failed to get attribute '%s' on '%s'" % (field_name, entity_name), e)
+            except Exception as e:
+                raise ExternalException(None, "Failed to get attribute '%s' on '%s'" % (field_name, entity_name), e)
```

Comment:
This improves the readability of the code:
```suggestion
            except RuntimeException as e:
                raise WrappingRuntimeException(None, f"Failed to get attribute '{field_name}' on '{entity_name}'", e)
            except plugins.PluginException as e:
                raise ExplicitPluginException(None, f"Failed to get attribute '{field_name}' on '{entity_name}'", e)
            except Exception as e:
                raise ExternalException(None, f"Failed to get attribute '{field_name}' on '{entity_name}'", e)
```